src/Microsoft.ML.StandardLearners/Standard/MultiClass/PairwiseCouplingTrainer.cs

```diff
@@ -213,6 +214,9 @@ public override TTransformer Fit(IDataView input)
         }
     }
 
+    /// <summary>
+    /// Contains the model parameters and prediction functions for Pkpd.
+    /// </summary>
     public sealed class PkpdModelParameters :
```

Comment:
>PkpdModelParameters [](start = 24, length = 19)

Maybe rename this one too, so that it does not use the contraction.

Reply:
Also could you rename Pkpd in the summary above?

---
In reply to: [260863966](https://github.com/dotnet/machinelearning/pull/2749#discussion_r260863966) [](ancestors = 260863966)